config/jobs/kubernetes/sig-network/sig-network-misc.yaml

```diff
@@ -17,6 +17,47 @@ presubmits:
         - --
         - "true"
 
+  kubernetes/kubernetes:
+  - name: pull-kubernetes-e2e-kubeadm-dind-ipv6
+    context: pull-kubernetes-e2e-kubeadm-dind-ipv6
+    max_concurrency: 8
+    skip_report: true
+    always_run: false
+    run_if_changed: '^(cmd/kubeadm|build/debs).*$'
+    rerun_command: "/test pull-kubernetes-e2e-kubeadm-dind-ipv6"
+    trigger: "(?m)^/test pull-kubernetes-e2e-kubeadm-dind-ipv6,?(\\s+|$)"
+    labels:
+      preset-service-account: "true"
+    spec:
+      containers:
+      - image: diverdane/kubekins-e2e:v20180924-703d9d3-master
```

Comment:
our image bump script (experiment/bump_e2e_image.sh) is going to break this on the next image cut

Reply:
@BenTheElder Is the bump script triggered automatically, or is it something that's manually triggered from time to time, or when needed? I'm just trying to minimize the turnaround time to get this running after the first merge.

Reply:
It's manual, but right now for example it's relatively frequent as we're starting to merge changes to kubetest etc. again post code-freeze.